models/organization/org_time/org_times_test.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package orgtime_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"code.gitea.io/gitea/models/db"
+	_ "code.gitea.io/gitea/models/issues"
+	"code.gitea.io/gitea/models/organization"
+	"code.gitea.io/gitea/models/unittest"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestMain sets up the testing environment specifically for testing org times.
+func TestMain(m *testing.M) {
+	unittest.MainTest(m, &unittest.TestOptions{
+		GiteaRootPath: filepath.Join("..", "..", ".."),
+		FixtureFiles: []string{
```

Comment:
The FixtureFiles here is unnecessary any more

Reply:
I don't know, if I take that `FixtureFiles` out, the tests fail with:

```
=== RUN   TestTimesPrepareDB
LoadFixtures failed after retries: testfixtures: could not clean table "action": no such table: action
    org_times_test.go:27:
                Error Trace:    /persist/gitea/src/models/organization/org_time/org_times_test.go:27
                Error:          Received unexpected error:
                                testfixtures: could not clean table "action": no such table: action
                Test:           TestTimesPrepareDB
--- FAIL: TestTimesPrepareDB (4.02s)
=== RUN   TestTimesByRepos
=== RUN   TestTimesByRepos/Full_sum_for_org_1
    org_times_test.go:104:
                Error Trace:    /persist/gitea/src/models/organization/org_time/org_times_test.go:104
                Error:          Received unexpected error:
                                user does not exist [uid: 1, name: , keyid: 0]
                Test:           TestTimesByRepos/Full_sum_for_org_1
--- FAIL: TestTimesByRepos (0.00s)
    --- FAIL: TestTimesByRepos/Full_sum_for_org_1 (0.00s)
```

Please advise!

Reply:
Since now almost all tests has a different package name with non-test code in the same directory. We can import the package where action is in.

Reply:
Thanks for the feedback, but sorry, I'm not sure I understand it. Yes, we can import the package ( `code.gitea.io/gitea/models/organization`, where the tested functions are, is already imported), but how does that help against the test data (fixtures) tables not existing for the tests, if I remove the FixtureFiles like you asked?

Reply:
Like you import `_ "code.gitea.io/gitea/models/issues"`, The init() function will `RegisterModel` the necessary model for this test package.